### PR TITLE
Speed up render

### PR DESCRIPTION
**Pull request by Jephuff**
This currently cuts initial render time by ~about~ over 50%

### CODE REVIEW

**Comment by shanewilson** [comment]
are you publishing to npm when you have those version commits?

**Comment by Jephuff** [comment]
They are published with the beta tag.

**Comment by shanewilson** [comment]
should be `1.4.0-beta.3` maybe then?

**Comment by Jephuff** [comment]
I've never loved adding things to the version, but yeah in this case that probably would have been a good idea :)

File: API.md [hunk 1]
```diff
@@ -125,8 +125,11 @@ for styling and the opacity function passed in for the tracks.
 * `removeDonors(function)` - Removes all donors that evaluate true for the passed function.
 * `removeGenes(function)` - Removes all genes that evaluate true for the passed function.
 * `toggleHeatmap()` - Toggle between heatMap and regular mode.
+* `setHeatmap()` - Set heatMap mode, accepts `true` or `false`.
```
Comment (andricDu) [inline API.md:128]: might want to specify the argument type here.

**Review by andricDu — CHANGES_REQUESTED** [review]

File: API.md (hunk 1 above)
Comment (Jephuff) [reply API.md:128]: ahh oops, I'll make a pr to fix that :)

**Pull request merged**